1.9/usage/debugging/cli-debugging/debug-subcommands/index.md

```diff
@@ -0,0 +1,64 @@
+---
+post_title: DC/OS CLI Debug Subcommands
+feature_maturity: experimental
+menu_order: 0
+---
+
+The DC/OS CLI provides a set of debugging subcommands, `dcos marathon debug`. A large portion of this functionality is also available from the DC/OS web interface.
+
+# Using dcos marathon debug
+
+## Prerequisites
+- A DC/OS cluster.
+- The DC/OS CLI installed.
+- A service or pod that is failing or stuck in deployment.
```

Comment:
I would remove `failing`. Reasonably you can only use `dcos marathon debug` to debug stuck deployments where marathon waits to the matching offer to occur.

If you have a failing app or pod it would imply that the container starts and fails directly afterwards (this is semantically `failing` in mesos/marathon world). In this particular scenario `dcos marathon debug` would not help, because this is build to analyze why marathon do not receive matching offers. In case of an failing app, marathon received matching offers, but could not start the container correctly on these offers.